src/Sylius/Component/Addressing/Model/AddressLogEntry.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Addressing\Model;
+
+use Sylius\Bundle\ResourceBundle\Model\ResourceLogEntry;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+final class AddressLogEntry extends ResourceLogEntry
```

Comment:
Unfortunately we must have this class, to have custom table for each loggable entity (we don't want to store, for example, product changes in the same table as address changes).